docker-compose-auditor.yml

```diff
@@ -76,12 +76,15 @@ services:
       - SCALAR_DL_LEDGER_AUDITOR_ENABLED=true
 
   scalar-auditor:
-    image: ghcr.io/scalar-labs/scalar-auditor:3.9.1
-    # image: 709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-auditor:3.9.1
+    image: ghcr.io/scalar-labs/scalardl-auditor-byol:3.9.1
     container_name: "scalardl-samples-scalar-auditor-1"
     volumes:
       - ./fixture/auditor.pem:/scalar/auditor.pem
       - ./fixture/auditor-key.pem:/scalar/auditor-key.pem
+      - ./fixture/auditor.properties.tmpl:/scalar/auditor/auditor.properties.tmpl
```

Comment:
I mount `auditor.properties.tmpl` to the container from this repository. In other words, we use the `.properties` file maintained in this repository instead of the default (included in the container) `.properties` file.

This is because I want to set a license key via an environment variable.

First, I thought to update the [origin `.propertis` file](https://github.com/scalar-labs/scalar/blob/v3.9.1/auditor/conf/auditor.properties.tmpl). However, it's used in other (e.g., Pay As You Go) container images.

I feel setting license key related properties in the Pay As You Go container images is a bit odd. So, I copied it for `scalardl-samples`.

(But, I think we can discuss it. It might be better to update the origin `.properties` file to avoid duplicated maintenance. Your opinions are welcome!)

Reply:
Does the PAYG image work with the licensing properties (but without actual setting values)? If they are just ignored, I think it’s not so weird since they are kinds of optional values (except for the BYOL image).

Reply:
@jnmt 

Thank you for your comment!

> Does the PAYG image work with the licensing properties (but without actual setting values)?

I think the PAYG container image just ignores the license key properties even if we set the license key in the PAYG container image.

If there is no problem, I think updating the origin `.properties` file is better since we can reduce the maintenance cost.

@feeblefakie 
What do you think?

(If there is no problem, I will update the origin `.properties` file and update this PR based on it!)

Reply:
I think avoiding duplicated maintenance is better. So, I will update the origin `.properties` file on the ScalarDL repository side.

Reply:
We (@jnmt and I) discussed and decided as follows:

- Keep `.properties` file on the `scalardl-samples` side.
  - This is because, if we update the origin `.properties` file, we have to release new patch version of ScalarDL to apply this PR on the `scalardl-samples` side. To apply this PR to `3.9` branch and `main (3.9.2)` branch on the `scalardl-samples` side, we will keep the `.properties` files in this PR.
- However, we will update the origin `.properties` file to reduce future maintenance costs.
  - This updates on the ScalarDL side will apply `3.9.3`, `3.10.x`, and `4.0.0`.
  - So, we will update `scalardl-samples` side again to remove duplicated `.properties` files after we will release `3.9.3` and `3.10.0` in the future.